doc/contributing/interacting-with-social.md

```diff
@@ -0,0 +1,6 @@
+# Interacting with social
```

Comment:
"Interacting with social" is jargony. Most people don't translate "social" to "social media" much less "the social media team".

How about "Requesting social media posts" or something like that? (Same for the name of the file.)

Reply:
Ooh, even better might be "Suggesting social media posts". @nodejs/social

Reply:
Happy to change the any to any of those, or whatever based on any more feedback fom social team.

Reply:
Maybe "Requesting social media posts or amplification" ?

Reply:
I think "amplification" is still pretty jargony and makes it needlessly verbose, but I wouldn't block that. It's certainly more clear, I think, than "Interacting with social".

Reply:
The title of the link that we got from the Foundation staff was `Node.js Social Amplication Request Guidelines` I could just use that minues the Node.js part which I think would be redundant.

Reply:
To be specific `social-amplification-request-guideliens.md`

Reply:
> The title of the link that we got from the Foundation staff was `Node.js Social Amplication Request Guidelines` I could just use that minus the Node.js part which I think would be redundant.

You could, but it's jargon. Jargon is OK for an internal document or a draft, but if we're publishing it widely, "social amplification" is a term likely to be unfamiliar to most readers. We should use terms that most people are likely to understand and that they might be searching for so they can find the right doc. If I were trying to find out how to get the nodejs Twitter account to retweet something, I might search for "social media" or "Twitter" or "posts" or "tweet", but I would be unlikely to search for "amplification" or "interacting with social".

Reply:
@Trott , I really don't feel strong about it just thought leveraging language that the social team had used could make sense. I'll change to `suggesting-social-media-posts.md`